framework/src/play/Logger.java

```diff
@@ -65,31 +72,36 @@ public static void init() {
             log4jConf = Logger.class.getResource(log4jPath);
         }
         if (log4jConf == null) {
-            Properties shutUp = new Properties();
-            shutUp.setProperty("log4j.rootLogger", "OFF");
-            PropertyConfigurator.configure(shutUp);
+            LoggerContext ctx = (LoggerContext) LogManager.getContext(false);
+            LoggerConfig loggerConfig = ctx.getConfiguration().getRootLogger();
+            loggerConfig.setLevel(org.apache.logging.log4j.Level.OFF);
```

Comment:
what does this line do?
Sets the default and lets the configuration define the each particular level?